dali/ft/filecopy.cpp

```diff
@@ -2597,6 +2597,13 @@ void FileSprayer::setTarget(IDistributedFile * target)
         unknownTargetFormat = false;
     else
     {
+        const char* separator = srcFormat.separate.get();
+        if (separator && (*separator == ',') && (*(separator+1) == NULL) )
```

Comment:
Shouldn't use NULL for a character...

Just use streq(separator, ",") is much clearer.